docs/update.md

```diff
@@ -8,7 +8,7 @@ If updating across major versions (e.g. <=0.1.0 to 1.x.x, 1.x.x to 2.x.x, ...),
 
 **Step 1:** Make sure all your extensions have versions compatible with the Flarum version you're trying to install. This is only needed across major versions (e.g. you probably don't need to check this if upgrading from v1.0.0 to v1.1.0, assuming your extensions follow recommended versioning). You can check this by looking at the extension's [Discuss thread](https://discuss.flarum.org/t/extensions), searching for it on [Packagist](http://packagist.org/), or checking databases like [Extiverse](https://extiverse.com). You'll need to remove (not just disable) any uncompatible extensions before updating. Please be patient with extension developers!
 
-**Step 2:** Take a look at your `composer.json` file. Unless you have a reason to require specific versions of extensions or libraries, you should set the version string of everything except `flarum/core` (and `flarum/NAME` bundled extensions) to `*`. Make sure `flarum/core` is NOT set to `*`. If you're targeting a specific version of Flarum, set `flarum/core` to that (e.g. `"flarum/core": "v0.1.0-beta.16`). If you just want the most recent version, use `"flarum/core": "^0.1.0"`.
+**Step 2:** Take a look at your `composer.json` file. Unless you have a reason to require specific versions of extensions or libraries, you should set the version string of everything except `flarum/core` to `*`. Make sure `flarum/core` is NOT set to `*`. If you're targeting a specific version of Flarum, set `flarum/core` to that (e.g. `"flarum/core": "v0.1.0-beta.16`). If you just want the most recent version, use `"flarum/core": "^1.0.0"`.
```

Comment:
While this is covered in this step, I'm not sure it's clear that people need to update the requirements for Flarum core extensions. Many people will still have those requirements set to `^0.1.0` when we release Flarum stable.